src/containerapp/azext_containerapp/_help.py

```diff
@@ -61,6 +61,18 @@
               --image my-app:v1.0 --environment MyContainerappEnv \\
               --secrets mysecret=secretvalue1 anothersecret="secret value 2" \\
               --secret-volume-mount "mnt/secrets"
+    - name: Create a container app from a new GitHub Actions workflow in the provided GitHub repository
```

Comment:
Please add new example to the final part of _help.py:
```
# containerapp create for preview
helps['containerapp create'] = """
```
same as 
`helps['containerapp update']`

Reply:
You mean an example for `--repo`? That would not be necessary because `--repo` is only introduced to the `create` command and not the `update` command.

Reply:
You can use command `az containerapp create -h` to check If the example can be seen.
 I think this cannot be seen beacuse this is another in the bottom of file, which overwrite current `containerapp create` you modify.
```
# containerapp create for preview 
helps['containerapp create'] = """
```

Reply:
Thanks for clarifying! Added these changes in my latest commit.